### ISSUE TITLE
Build failure on Aarch64/Linux target

### ISSUE THREAD

**Issue by jserv** [LuisHsu/WasmVM#22]
While building on Aarch64/Linux, cmake complained as following:
```
[ 13%] Building CXX object src/Syscall/CMakeFiles/WasmVM_Syscall.dir/Call20.cpp.o
/tmp/WasmVM/src/Syscall/Call20.cpp: In function ‘void Call::sysFstat(Store&, Stack&)’:
/tmp/WasmVM/src/Syscall/Call20.cpp:226:33: error: no match for call to ‘(Stack) ()’
   Value* fd = (Value*)coreStack().data;
                                 ^
/tmp/WasmVM/src/Syscall/Call20.cpp:243:32: error: ‘pathnameAddr’ was not declared in this scope
   char* bufPtr = memoryData += pathnameAddr->data.i32;
                                ^~~~~~~~~~~~
/tmp/WasmVM/src/Syscall/Call20.cpp: In function ‘void Call::sysSelect(Store&, Stack&)’:
/tmp/WasmVM/src/Syscall/Call20.cpp:550:50: error: ‘data’ was not declared in this scope
   char* readfdsPtr = memoryData += readfdsAddr > data.i32;
                                                  ^~~~
```

Extra information:
* `g++ -v` dumps `gcc version 6.3.0`.

**Comment by LuisHsu** [comment]
Thanks!
Error fixed at `master` [#e2b62b8](https://github.com/LuisHsu/WasmVM/commit/e2b62b8c1fe3f7f5bde0bbea72a4e3dc772ed857)

**Comment by jserv** [comment]
Confirmed on aarch64-linux.
